### PR TITLE
docs: Some small fixes

### PR DESCRIPTION
**Pull request by yagebu**
The commit messages are hopefully descriptive enough.

### CODE REVIEW

**Comment by anarcat** [comment]
otherwise this all looks like good fixes, thanks!

File: docs/installation.rst [hunk 1]
```diff
@@ -190,19 +188,17 @@ This uses the latest (source package) release from PyPi.
...
 This uses latest, unreleased development code from git.
-While we try not to break master, there are no guarantees on anything.
-
-.. parsed-literal::
+While we try not to break master, there are no guarantees on anything. ::
 
-    # get |project_name| from github, install it
-    git clone |git_url|
+    # get borg from github
+    git clone https://github.com/borgbackup/borg.git
```
Comment (ThomasWaldmann) [inline docs/installation.rst]: no, don't replace the placeholders with hardcoded text.

File: setup.py [hunk 2]
```diff
@@ -184,8 +184,8 @@ def run(self):
         print("auto-generating API documentation")
         with open("docs/api.rst", "w") as doc:
             doc.write("""
-Borg Backup API documentation
-=============================
+API Documentation
```
Comment (ThomasWaldmann) [inline setup.py]: good catch. :)

File: docs/installation.rst (hunk 1 above)
Comment (yagebu) [inline docs/installation.rst]: This broke the html output (this code block looks uglier than the other ones...)

Comment (yagebu) [inline docs/installation.rst]: Currently, the placeholders are not used very consistently anyway, even before this change there were 14 mentions of 'borg' in `installation.rst` alone. And even one linking to github...

Comment (ThomasWaldmann) [inline docs/installation.rst]: that might be, but the correct fix then is to use the placeholders, if possible.

Comment (ThomasWaldmann) [inline docs/installation.rst]: @yagebu can you define "broke" more precisely?

Comment (yagebu) [inline docs/installation.rst]: Sphinx doesn't do any highlighting for the `parsed-literal` block and the CSS style for non-highlighted blocks is different

docs

Comment (yagebu) [inline docs/installation.rst]: I've asked on the `sphinx_rtd_theme` to see if this is intentional

Comment (anarcat) [inline docs/installation.rst]: can you link to the issue directly?

Comment (yagebu) [inline docs/installation.rst]: https://github.com/snide/sphinx_rtd_theme/issues/65#issuecomment-147772411

**Pull request merged**